src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -455,8 +456,8 @@ class JesBackend extends Backend with LazyLogging with ProductionJesAuthenticati
 
     jesJobSetup match {
       case Failure(ex) =>
-        log.error(s"Failed to create a JES run", ex)
-        FailedExecutionHandle(ex)
+        log.warn(s"Failed to create a JES run", ex)
+        throw ex  // Probably a transient issue, throwing retries it
```

Comment:
Same comment as above re throwing exception as an indicator for "retry me please" feeling strange.